libs/features/game-tele/src/edit-game-tele/game-tele.integration.spec.ts

```diff
@@ -0,0 +1,178 @@
+import { ComponentFixture, TestBed, waitForAsync } from '@angular/core/testing';
+import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import { GameTele } from '@keira/shared/acore-world-model';
+import { MysqlQueryService, SqliteService } from '@keira/shared/db-layer';
+import { EditorPageObject, TranslateTestingModule } from '@keira/shared/test-utils';
+import { ModalModule } from 'ngx-bootstrap/modal';
+import { ToastrModule } from 'ngx-toastr';
+import { of } from 'rxjs';
+import { instance, mock } from 'ts-mockito';
+import { GameTeleComponent } from './game-tele.component';
+import Spy = jasmine.Spy;
+import { GameTeleHandlerService } from '../game-tele-handler.service';
+
+describe('GameTele integration tests', () => {
+  // Page Object Class
+  class GameTelePage extends EditorPageObject<GameTeleComponent> {}
```

Comment:
no need for the comment 

```suggestion
  class GameTelePage extends EditorPageObject<GameTeleComponent> {}
```